src/WorkerPool.js

```diff
@@ -7,6 +7,7 @@ import asyncMapSeries from 'neo-async/mapSeries';
 
 import readBuffer from './readBuffer';
 import WorkerError from './WorkerError';
+import { replacer, reviver } from './serializer';
```

Comment:
Rather than using a custom serializer, you should use the Node serialization APIs:
https://nodejs.org/api/v8.html#v8_serialization_api

Reply:
I prefer using a more familiar and portable API as long as it solves the same problem.

@alexander-akait what do you think about this?

Reply:
v8 seriliazer doesn't support many feature